### DISCUSSION TITLE
Is it ready for production on Android and iOS?

### DISCUSSION THREAD

**Discussion by fzyzcjy** [tomaka/cpal#1324] [Q&A]
Hi thanks for the library! I wonder whether it is ready for production environment on Android and iOS?

**Comment by setoelkahfi** [comment]
I manage to use this crate through `rodio` and play audio playback. But the audio is so low on Android and iOS. And on Android, ~~it lists duplicate available hosts.~~

**Comment by roderickvd** [comment]
Please try and let us know how you find it! 😄 

Half-joking though: being pre-1.0 we make no assertions to fitness for use. Even when hitting 1.0 I'm not sure if I'll ever say that, because production-ready for _what?_

We hold correctness, robustness and quality in high regard, and will be happy to work with you on any issues reported.
